server/lib/notifications.js

```diff
@@ -214,6 +215,10 @@ async function notifyByEmail(activity) {
 
     case activityType.COLLECTIVE_MEMBER_CREATED:
       twitter.tweetActivity(activity);
+      if (activity.data.member.CreatedByUserId) {
+        const userAdmin = await models.User.findByPk(activity.data.member.CreatedByUserId);
```

Comment:
`CreatedByUserId` should not be used because there is no guarantee that this what you need here.

You should look at the `memberCollective` and query the User table with its `CollectiveId`

Reply:
There is also no guarantee that the member is an user/individual and that userAdmin will exist, you should take that into account.